src/routers/employer.ts

```diff
@@ -17,5 +18,5 @@ router.post('/jobs', tokenVerify, createJobPost)
 router.patch('/jobs/:id', tokenVerify, updateJobPost)
 router.patch('/', tokenVerify, updateEmployer)
 router.delete('/jobs/:id', tokenVerify, deleteJobPostbyId)
-
+router.post('/upload/profile', tokenVerify, uploadImages)
```

Comment:
If both `jobseeker` and `employer` can use this route. Perhaps you can rename the login router as `commonRouter` where both user types can use it and put `uploadImages` controller in there.